emspy/query/fltquery.py

```diff
@@ -138,13 +142,17 @@ def __translate_expr(self, expr_vec):
 		fld_loc = [False, False]
 		fld_info = None
 		fld_type = None
-		val_info = None
-		op = expr_vec[1]
+		val_info = []
+
+		if len(expr_vec) == 3:
```

Comment:
What are 3 and 5?  I'm guessing this has to do with the added "between" filtering . Can you include some comments on that here?

Reply:
I added inline comments for this in c30d338